hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -309,6 +309,9 @@ def do_draw(self, data):
         reference = data.draw(self.__reference_strategy)
         return machine.names_to_values[reference.name]
 
+    def __repr__(self):
+        return "<Bundle name=%s>" % (self.name,)
```

Comment:
The syntax for this is a little weird. Why not `Bundle(name=%s)`?

Reply:
... I think just path-dependence in the pile of stuff I was trying out at the time.  Will change.